### DISCUSSION TITLE
OSError on RaspberryMatic-HmIP-RF

### DISCUSSION THREAD

**Discussion by Mark-HomeAssistantTC** [SukramJ/aiohomematic#1043] [Q&A]
Hallo zusammen,
ich erhalte im Home Assistant OS folgende Fehlermeldung in den Logs:

Logger: hahomematic.xml_rpc_proxy
Source: custom_components/homematicip_local/control_unit.py:768
Integration: Homematic(IP) Local ([documentation](https://github.com/danielperna84/custom_homematic), [issues](https://github.com/danielperna84/hahomematic/issues))
First occurred: 13:46:41 (87 occurrences)
Last logged: 20:56:41

OSError on RaspberryMatic-HmIP-RF: (8, 'EOF occurred in violation of protocol (_ssl.c:2396)')

Was sagt mir dieser Fehler und wie kann ich ihn beheben?
Eigentlich funktionieren alle (physischen) Geräte und ich kann diese sowohl über Home Assistant als auch über das Raspberrymatic Addon steuern.
Nur die virtuellen Kanäle funktionieren nicht.

Viele Grüße

**Comment by houseio** [comment]
Running on " IntelNUC8Gen.i5-8259U(16GB-RAM/512GB), Home Assistant 2023.8.2 OSystem 10.4 " 
 : Same problem here:

 Logger: hahomematic.xml_rpc_proxy
Source: runner.py:179
First occurred: 2:16:05 PM (7 occurrences)
Last logged: 3:55:41 PM

    OSError on RaspberryMatic-HmIP-RF: (8, 'EOF occurred in violation of protocol (_ssl.c:2423)')
    OSError on RaspberryMatic-BidCos-RF: (8, 'EOF occurred in violation of protocol (_ssl.c:2423)'

**Reply by SukramJ** [reply]
> Was sagt mir dieser Fehler und wie kann ich ihn beheben?

Da der Fehler sehr sporadisch auftritt, wäre das einfachste TLS zu deaktivieren.
